inc/tai.h

```diff
@@ -1,145 +1,192 @@
 /**
- * Copyright (c) 2018 Nippon Telegraph and Telephone Corporation.
+ * @file    tai.h
+ * @brief   This module defines an entry point into Transponder Abstraction 
+ *          Interface (TAI)
  *
- * This source code is licensed under the BSD 3-Clause license found in the
- * LICENSE file in the root directory of this source tree.
+ * @copyright Copyright (c) 2014 Microsoft Open Technologies, Inc.
```

Comment:
I consider this a derivative work from SAI. It would be hard to say that this is all new, and not based on SAI. So I kept the Microsoft copyright. Also, I'm not a lawyer so it's not clear to me if we should change the licensing from Apache v2 to BSD 3-Clause. Is that allowed?

Reply:
Hmm, I'm also not sure changing license for Apache v2 is allowed.

As for TIP official git repository, I had to choose the license from BSD 3-Clause or a limited copyright-only license.
If we can keep TAI's license Apache v2, that would be the easiest.

@mikeypiro Can I change the license to Apache v2?